src/main/java/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java

```diff
@@ -81,11 +81,7 @@ public void onCreate(Bundle paramBundle) {
       autoDelServer.setOnPreferenceChangeListener(new AutodelChangeListener("delete_server_after"));
     }
     if (dcContext.isChatmail()) {
-      autoDelServer.setEntries(new CharSequence[]{getString(R.string.automatic), getString(R.string.autodel_at_once)});
```

Comment:
"automatic" string can probably be deprecated once not used on other platforms? Not sure how deprecation works in this case when you need to track if it is still used by other platforms. It can be mentioned at least in the translator comments that the string is not going to be used anywhere so they don't try to find what is the context for this string.